### DISCUSSION TITLE
Sensor FSL 2 not detected by xDrip anymore

### DISCUSSION THREAD

**Discussion by Sarrazin87** [NightscoutFoundation/xDrip#2053] [Q&A]
Hi everyone,
I need your help, please!!
I've been using xDrip+ with freestyle libre 2 for at least 2 months now and never got any problem. Until today!
I changed the sensor, scanned it with librelink patche to start it, as usual. And then stopped and started it on xDrip, as asual too. And nothing! xDrip doesn't seem to connect to it anymore. I changed the sensor 6 hours ago and absolutely nothing from xDrip. I restarted my phone (samsung A52s), still nothing. Thus, AAPS doesn't receive anything. I'm in the process to the closed loop. I'm on level 4 of the objectives, and it's my day two. So I'm afraid this problem lasts and doesn't allow me to pass it. 
I feel lost. I was so used to see how my BG was going to be, and now I don't see anything.
I keep reading in xDrip the following sentence:
"Aggressively restarting collector service due to lack of reception : backoff: 240" (the numbers change, sometimes it's 120, 180, 300 or even 360).

Did it happened to any of you? What did you do?
Sorry for the long post and thanks for your help.
Sarra

**Comment by tzachi-dar** [comment]
Please make sure it is a libre 2 sensor.
Can you export your db and send it to tzachi.dar@gmail.com

**Reply by Sarrazin87** [reply]
Problem solved!
The sensor had a default since I wasn't receiving any alert when having a high or a low BG.
I changed it, and now everything's back to normal. Thanks for your help.

**Reply by LuisPuertollano** [reply]
Hi @Sarrazin87 , I´m having I think the same problem...was your FSL2 sensor expiration date something in the near future o do you know the S/N from it? I think I´ll change it right now and call support tomorrow.

**Reply by LuisPuertollano** [reply]
![Screenshot_20220323-204527_xDrip+](https://user-images.githubusercontent.com/77344100/159795560-18404cd8-152d-4c5d-9b7d-faf6cec5a54f.jpg)
![Screenshot_20220323-204531_xDrip+](https://user-images.githubusercontent.com/77344100/159795565-a85e3dd2-a325-406b-be2f-cb9f86ed084a.jpg)
![Screenshot_20220323-203013_xDrip+](https://user-images.githubusercontent.com/77344100/159795569-288f77b2-8484-464a-b2ee-5d82fb99eb09.jpg)

**Reply by Sarrazin87** [reply]
Hello,
It doesn't seem to be the exact same problem since my version had the following error message : "Aggressively restarting collector service due to lack of reception : backoff: 240"
I think my sensor didn't have a Bluetooth connection, as if it was a Libre 1 and not a Libre 2 (but it was really a 2). So I changed it after calling the Libre support on Monday and it worked perfectly well after that (and I received the new sensor from Libre support this morning! They're so fast!). I hope yours will be working again after changing it 🤞🏼

**Reply by LuisPuertollano** [reply]
well, I´m triying anyway...my "maybe-defect-sensor" didn´t gave me any confidence,...I remember having a similar problem in the past (maybe 1 year ago?), which got fixed after changing the sensor...I hope it really works...I´m glad with FSL2 but with FSL3 i don´t think Abbott is really future-proof. I don´t know, personal opinion.
I´ll answer here in 1 hour :-) Thank you!

**Comment by Sarrazin87** [comment]
Yes, let us know if it works after changing it.
I want to change for the Dexcom G6. I hope my new Doctor will approve 🤞🏼

**Reply by LuisPuertollano** [reply]
well, after 1h warm-up, it seems to work again. Bad sensor I guess.
But yeah, G6 seems also really good. thank you!

**Reply by Sarrazin87** [reply]
That's great! Thanks for letting us know 😁
